test/test_t2ramsey.py

```diff
@@ -227,10 +225,11 @@ def test_t2ramsey_parallel(self):
         t2ramsey = [30, 25]
         estimated_freq = [0.1, 0.12]
         delays = [list(range(1, 60)), list(range(1, 50))]
-        dt_factor = 1
+        dt_factor = 1e-6
```

Comment:
What's the origin of this change?

Reply:
I added this just to make the numbers more realistic (microsec rather than sec)